src/main/java/org/mockbukkit/mockbukkit/inventory/ItemStackMock.java

```diff
@@ -212,8 +264,8 @@ public static ItemStackMock empty()
 		ItemStackMock clone = new ItemStackMock(this.type);
 
 		clone.setAmount(this.amount);
+		clone.setItemMeta(this.itemMeta == null ? null : this.itemMeta);
```

Comment:
Yout don't need to do a ternary statement here
```suggestion
clone.setItemMeta(this.itemMeta);
```